### PR TITLE
Update Changes

### PR DESCRIPTION
**Pull request by delbegum1**

### CODE REVIEW

File: src/test/java/Com/FirstMavenProject/FirstClassTestNG.java [hunk 1]
```diff
@@ -16,11 +16,11 @@ public void firstMethod() throws InterruptedException   {
 System.setProperty("webdriver.chrome.driver", "Driver/chromedriver");  
 		
 		WebDriver driver=new ChromeDriver();  
-		
+		Thread.sleep(2000);
```
Comment (ImMayankGupta) [inline src/test/java/Com/FirstMavenProject/FirstClassTestNG.java]: Please change time to 3000

File: src/test/java/Com/FirstMavenProject/FirstClassTestNG.java [hunk 2]
```diff
@@ -16,11 +16,11 @@ public void firstMethod() throws InterruptedException   {
...
 		//driver.get("https://google.com");//
 		
 		driver.navigate().to("http://www.yahoo.com/");
-		
+		Thread.sleep(2000);
 		driver.findElement(By.id("ybar-sbq")).sendKeys("New York");
 		Thread.sleep(2000);
```
Comment (Rony199430) [inline src/test/java/Com/FirstMavenProject/FirstClassTestNG.java]: change the time to 5 seconds.

File: src/test/java/Com/FirstMavenProject/FirstClassTestNG.java [hunk 3]
```diff
@@ -16,13 +16,13 @@ public void firstMethod() throws InterruptedException   {
 System.setProperty("webdriver.chrome.driver", "Driver/chromedriver");  
 		
 		WebDriver driver=new ChromeDriver();  
-		
+		Thread.sleep(3000);
 		//driver.get("https://google.com");//
```
Comment (delbegum1) [inline src/test/java/Com/FirstMavenProject/FirstClassTestNG.java:20]: test

File: src/test/java/Com/FirstMavenProject/FirstClassTestNG.java (hunk 1 above)
Comment (delbegum1) [reply src/test/java/Com/FirstMavenProject/FirstClassTestNG.java]: Update complete

Comment (ImMayankGupta) [reply src/test/java/Com/FirstMavenProject/FirstClassTestNG.java]: Thanks

**Review by ImMayankGupta — APPROVED** [review]

**Review by Rony199430 — APPROVED** [review]

**Pull request merged**